sdk/schemaregistry/schema-registry-avro/test/errors.spec.ts

```diff
@@ -14,6 +15,30 @@ import { v4 as uuid } from "uuid";
 
 chaiUse(chaiPromises);
 
+async function assertError<T>(
+  p: Promise<T>,
+  expectations: {
+    innerMessage?: RegExp;
+    message?: RegExp;
+  } = {}
+): Promise<void> {
+  const { innerMessage, message } = expectations;
+  try {
+    await p;
```

Comment:
what if `p` resolves correctly?

```suggestion
    await p;
    assert.fail("Expected promise to error, but resolved successfully");
```